Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -949,13 +945,6 @@ PhysicalParticleContainer::Evolve (int lev,
     WARPX_PROFILE_VAR_NS("PPC::EvolveOpticalDepth", blp_ppc_qed_ev);
     WARPX_PROFILE_VAR_NS("PPC::ParticlePush", blp_ppc_pp);
 
-    const std::array<Real,3>& dx = WarpX::CellSize(lev);
-    const std::array<Real,3>& cdx = WarpX::CellSize(std::max(lev-1,0));
-
-    // Get instances of NCI Godfrey filters
-    const auto& nci_godfrey_filter_exeybz = WarpX::GetInstance().nci_godfrey_filter_exeybz;
-    const auto& nci_godfrey_filter_bxbyez = WarpX::GetInstance().nci_godfrey_filter_bxbyez;
-
```

Comment:
Fixes the warnings:
`./Source/Particles/PhysicalParticleContainer.cpp:936:31`: warning: unused variable ‘dx’ [-Wunused-variable]
     const std::array<Real,3>& dx = WarpX::CellSize(lev);
`./Source/Particles/PhysicalParticleContainer.cpp:937:31`: warning: unused variable ‘cdx’ [-Wunused-variable]
     const std::array<Real,3>& cdx = WarpX::CellSize(std::max(lev-1,0));
`./Source/Particles/PhysicalParticleContainer.cpp:940:17`: warning: unused variable ‘nci_godfrey_filter_exeybz’ [-Wunused-variable]
     const auto& nci_godfrey_filter_exeybz = WarpX::GetInstance().nci_godfrey_filter_exeybz;
`./Source/Particles/PhysicalParticleContainer.cpp:941:17`: warning: unused variable ‘nci_godfrey_filter_bxbyez’ [-Wunused-variable]
     const auto& nci_godfrey_filter_bxbyez = WarpX::GetInstance().nci_godfrey_filter_bxbyez;